app/assets/javascripts/oxalis/model/bucket_data_handling/texture_bucket_manager.js

```diff
@@ -134,7 +136,7 @@ export default class TextureBucketManager {
   }
 
   keepLookUpBufferUpToDate() {
-    if (this.isRefreshBufferOutOfDate) {
+    if (this.lookUpTexture.isInitialized() && this.isRefreshBufferOutOfDate) {
```

Comment:
As discussed, it would be better to move this check into the `startRAFLoops` method and wait for the initialization of the texture, there :) Same for the check in the `processWriterQueue` method.